receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -269,15 +269,23 @@ func addHistogramDatapoints(_ pmetric.ResourceMetrics, _ labels.Labels, _ writev
 // addDatapoints adds the labels to the datapoints attributes.
 // TODO: We're still not handling several fields that make a datapoint complete, e.g. StartTimestamp,
 // Timestamp, Value, etc.
```

Comment:
Maybe remove this TODO as you are addressing it here
```suggestion
```

Reply:
Can the `StartTimestamp` be set like this?

`dp.SetStartTimestamp(pcommon.NewTimestampFromTime(time.Now()))`

It looks like we might not need to manually convert it to nanoseconds since `NewTimestampFromTime` already does that:

```Go
return Timestamp(uint64(t.UnixNano()))
```

Reply:
Actually, Prometheus samples only hold timestamp and value. StartTimeTimestamp comes from a separate field in the remote write, called CreatedTimestamp.

Let's tackle StartTimeTimestamp in a separate PR and focus only in timestamp and value here!

Small PRs are easier to review and get them ready to merge :)

Reply:
Thanks. Just to clearing up this doubt here, because it's in the same context, my idea was to open a new PR to address the lack of StartTimestamp

Reply:
Yeah, StartTimeTimestamp should be a new PR, and it is populated from the field Created Timestamp in the remote write request.